fe/fe-core/src/main/java/org/apache/doris/datasource/hive/HiveMetaStoreCache.java

```diff
@@ -171,6 +173,20 @@ public Map<PartitionCacheKey, HivePartition> loadAll(Iterable<? extends Partitio
 
                 });
 
+        fileSystemCache = CacheBuilder.newBuilder().maximumSize(Config.max_hive_partition_cache_num)
```

Comment:
It is strange to use `Config.max_hive_partition_cache_num` as the max size of fs cache.